Cargo.toml

```
@@ -23,10 +23,12 @@ bp-asset-hub-polkadot = { path = "system-parachains/asset-hubs/asset-hub-polkado
 bp-bridge-hub-cumulus = { version = "0.19.0", default-features = false }
 bp-bridge-hub-kusama = { path = "system-parachains/bridge-hubs/bridge-hub-kusama/primitives", default-features = false }
 bp-bridge-hub-polkadot = { path = "system-parachains/bridge-hubs/bridge-hub-polkadot/primitives", default-features = false }
+bp-people-hub-polkadot = { path = "system-parachains/people/people-polkadot/bridge-primitives", default-features = false }
 bp-header-chain = { version = "0.19.0", default-features = false }
 bp-messages = { version = "0.19.0", default-features = false }
 bp-parachains = { version = "0.19.0", default-features = false }
 bp-polkadot-core = { version = "0.19.0", default-features = false }
+bp-polkadot-bulletin = { path = "system-parachains/people/bulletin-polkadot/bridge-primitives", default-features = false }
```

Let's change this path little bit, or we can put it inside `system-parachains/people/people-polkadot/bridge-primitives`

```suggestion
bp-polkadot-bulletin = { path = "system-parachains/people/people-polkadot/bulletin-primitives", default-features = false }
```

---

I put it here thinking we can put the bulletin runtime here in the end. So the bridge primitives will be with the runtime.